test/other/test_parseTools.js

```diff
@@ -25,10 +25,21 @@ mergeInto(LibraryManager.library, {
     return 0;
   },
 
+  test_makeGetValue__deps: ['$readI53FromI64', '$readI53FromU64'],
   test_makeGetValue: function(ptr) {
     out('\ntest_makeGetValue:');
     let val;
 
+    // i53
+    val = {{{ makeGetValue('ptr', '0', 'i53') }}};
+    out('i53: ' + val.toString(16))
+    assert(val == -0xaabb12345678);
+
+    // u53
+    val = {{{ makeGetValue('ptr', '0', 'u53') }}};
+    out('u53: ' + val.toString(16))
+    assert(val == 0xffff5544edcba800);
```

Comment:
Is this right? It looks like it's a little rounded. But the idea of i53 is that we avoid rounding?

Reply:
Hmm .. this is existing behaviour which I'm not changing as part of this PR.   This is `makeGetValue` which calls `readI53FromU64` .. i guess maybe there is bug in that existing code.   Investigating.

Reply:
The issue is that the bytes in memory are `0xffff5544edcba988` which is withing `isSafeInteger` range when interpreted as a signed value, but not when interpreted as an unsigned value.  I added a comment.  There is no behaviour change here, just adding testing.

Reply:
I see. So as an i53 it's ok, but as a u53 it isn't. Maybe we should disallow u53..? But, separate from this PR, as you said.

Reply:
Even with i53 you can give it something out-of-range.   The idea with i53 is that things outside of the range not allowed/expected.   For write i53 we have very assertions and checked version of the helper functions, but currently the reader functions are silent.  We should fix that: https://github.com/emscripten-core/emscripten/blob/9fdc94ad3e3c89558fd251048e8ae2c2ca408dc1/src/library_int53.js#L82-L87